### ISSUE TITLE
[Programmatic Access – Make Code – Home Page] Role is not defined for all tiles present under ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3548]
**User Experience:**
User would find difficulty to identify the controls if role is not defined for that control as user won't understand the purpose of that control and also he/she would surely miss the functionality over that control.

**Note**: User credentials should NOT be included in the bug. 

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that role is defined for tiles control like(Flashing Heart, Name tag, Smiley Buttons, Dice, Love Meter, Micro Chat, Coin Toss, Step counter, Snap the dot, Coin flipper, 7 second etc.)

**Actual Result:**
Role is not defined for tiles control like(Flashing Heart, Name tag, Smiley Buttons, Dice, Love Meter, Micro Chat, Coin Toss, Step counter, Snap the dot, Coin flipper, 7 second etc.) under ‘Home page’ and screen reader is reading as group and due to this user won't be able to identify the control and won't be perform functionality over that control. 

**Expected Result:**
Role should be define as 'Button' for tiles control like(Flashing Heart, Name tag, Smiley Buttons, Dice, Love Meter, Micro Chat, Coin Toss, Step counter, Snap the dot, Coin flipper, 7 second etc.) under ‘Home page’, so that user could easily identify the control and take an action accordingly.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 4.1.2 – Name, Role, Value ](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)

![WCAG 4 1 2_Role is not defined](https://user-images.githubusercontent.com/72871089/97850712-696c6a00-1d1a-11eb-8baa-9d9f6022656e.jpg)
